package.json

```diff
@@ -1,6 +1,6 @@
 {
 	"name": "eslint-plugin-unicorn",
-	"version": "50.0.1",
+	"version": "51.0.0",
```

Comment:
You should never bump a version in a pull request. It's up to the maintainers to decide when and what version to use.